Gemfile

```diff
@@ -42,6 +42,8 @@ group :development, :test do
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '~> 2.13'
   gem 'selenium-webdriver'
+  gem 'debase'
+  gem 'ruby-debug-ide'
```

Comment:
Why you add this gem which in only for IDEs?

Reply:
https://rubygems.org/gems/ruby-debug-ide/versions/0.6.0

Reply:
Can I try to merge

Reply:
This gem its only for IDEs like I said. It's not okay to be in the GEM file because no one use IDE, except you I think. Need to be only in your GEM file.

Reply:
This is a gem for development purpose I do not see why not ?

Reply:
And i need the binstubs also

Reply:
@pau4o Can you take a look on the PR 
@ElitsaPetrova is not ok with my PR cause she says that I must have all this locally because noone else dosnt need it

Reply:
I desagree with that

Reply:
I talk only for the gem `ruby-debug-ide`.